### PR TITLE
init LogHandleHeader

### PR DESCRIPTION
**Pull request by Jazz86**

### CODE REVIEW

File: src/Header/ErrorHandle.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class ErrorHandle
+{
+public:
+    //singleton code convention method.
+    static ErrorHandle *GetInstance();
+    //non copyable
+    ErrorHandle(ErrorHandle &cpy) = delete;
+    ErrorHandle &operator=(ErrorHandle &cpy) = delete;
+    //record these infomation of error event
+    static int recordError(std::string time, std::string ip, std::string user, std::string statusCode, std::string message);
```
Comment (john0312) [inline src/Header/ErrorHandle.h]: const ref

File: src/Header/ErrorHandle.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_ErrorHandle_H_
+#define GROUP4_HEADER_ErrorHandle_H_
+
+#include <string>
+
+// write error message to local file
+class ErrorHandle
```
Comment (john0312) [inline src/Header/ErrorHandle.h]: Consider moving other logging code into this class, and call this class LogHandle or similar.

File: src/Header/ErrorHandle.h (hunk 1 above)
Comment (Jazz86) [reply src/Header/ErrorHandle.h]: done.

File: src/Header/ErrorHandle.h (hunk 2 above)
Comment (Jazz86) [reply src/Header/ErrorHandle.h]: done.

**Pull request merged**